### DISCUSSION TITLE
Image files without license information

### DISCUSSION THREAD

**Discussion by rahulmohang** [cypress-io/cypress#28381] [General]
Hi, 

The following image contains the copyright statement, 'Copyright 2007 Apple Inc., all rights reserved':

https://github.com/cypress-io/cypress/blob/develop/system-tests/projects/todos/tests/_fixtures/images/sample.tif

and 

the following images contain the copyright statement, 'Copyright (c) 1998 Hewlett-Packard Company':

https://github.com/cypress-io/cypress/blob/develop/system-tests/projects/e2e/cypress/fixtures/sample.jpg
https://github.com/cypress-io/cypress/blob/develop/system-tests/projects/todos/tests/_fixtures/images/sample.jpg

Statements on license of the files could not be found. Can you please let us know the license under which the files may be used?

**Comment by ElectroBoy404NotFound** [comment]
Those are the copyright statements on every image of those formats. They can't be avoided and also they don't need a licence to be used
